package.json

```diff
@@ -16,32 +15,26 @@
   "author": "João Nuno Silva <jnss81@gmail.com> (http://jnuno.com)",
   "main": "index.js",
   "scripts": {
-    "test": "istanbul cover _mocha",
-    "cov": "cat ./coverage/lcov.info | coveralls",
-    "lint": "jshint index.js test/test.js && eslint index.js test/test.js && npm run dtslint",
+    "test": "mocha",
+    "lint": "eslint index.mjs test/test.mjs",
     "lint:prettier": "prettier --check *.mjs test/*.mjs",
-    "preversion": "npm test && npm run lint && npm run dist && git add dist/TreeModel*.js",
+    "preversion": "npm test && npm run lint && git add dist/TreeModel*.js",
     "postversion": "git push && git push --tags && echo 'Dont forget to publish to npm...'",
     "dist": "rm -rf dist && mkdir -p dist && browserify index.js -o dist/TreeModel.js -s TreeModel && uglifyjs dist/TreeModel.js > dist/TreeModel-min.js",
```

Comment:
`dist` is calling uglifyjs which you removed in this PR

Reply:
removed git actions... but generally, these scripts/tasks will need to be reworked once we are closer to a release-worthy thing...